tests/test_openclhpp.cpp

```diff
@@ -3035,5 +3068,99 @@ void testDevicePCIBusInfo_KHR()
 #endif
 }
 
+/****************************************************************************
+ * Tests for cl::CommandBufferKhr
+ ****************************************************************************/
+#if defined(cl_khr_command_buffer)
+void testMoveAssignCommandBufferKhrNonNull();
+void testMoveAssignCommandBufferKhrNull();
+void testMoveConstructCommandBufferKhrNonNull();
+void testMoveConstructCommandBufferKhrNull();
+MAKE_MOVE_TESTS(CommandBufferKhr, make_command_buffer_khr, clReleaseCommandBufferKHR, commandBufferKhrPool);
+#else
+void testMoveAssignCommandBufferKhrNonNull() {}
+void testMoveAssignCommandBufferKhrNull() {}
+void testMoveConstructCommandBufferKhrNonNull() {}
+void testMoveConstructCommandBufferKhrNull() {}
+#endif
+
+// Stub for clGetCommandBufferInfoKHR that returns 1
+static cl_int clGetCommandBufferInfoKHR_testCommandBufferKhrGetNumQueues(
+    cl_command_buffer_khr command_buffer,
+    cl_command_buffer_info_khr param_name,
+    size_t param_value_size,
+    void *param_value,
+    size_t *param_value_size_ret,
+    int num_calls)
+{
+    (void) num_calls;
```

Comment:
If you do not use some parameters, do not name them
```suggestion
    int /* num_calls */)
{
```

Reply:
Sure, done.